app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -699,13 +746,17 @@ private fun mediaPlayerPlayButton() =
             .text("Play")
     )
 
-fun clickTabCrashedRestoreButton() {
-
-    assertTrue(
-        mDevice.findObject(UiSelector().resourceId("$packageName:id/restoreTabButton"))
-            .waitForExists(waitingTime)
+private var progressBar =
+    mDevice.findObject(
+        UiSelector().resourceId("$packageName:id/mozac_browser_toolbar_progress")
     )
 
-    val tabCrashRestoreButton = mDevice.findObject(UiSelector().resourceIdMatches("$packageName:id/restoreTabButton"))
-    tabCrashRestoreButton.click()
-}
+// Permissions test page elements & prompts
+// Test page used located at https://mozilla-mobile.github.io/testapp/permissions
+private val cameraButton = mDevice.findObject(UiSelector().text("Open camera"))
+
+private val microphoneButton = mDevice.findObject(UiSelector().text("Open microphone"))
+
+private val notificationButton = mDevice.findObject(UiSelector().text("Open notifications dialogue"))
+
+private val getLocationButton = mDevice.findObject(UiSelector().text("Get Location"))
```

Comment:
Are there resource strings for these locators?

Reply:
These are strings from the HTML page.